### PR TITLE
Adding "AuditLog" Support (WIP)

### PR DESCRIPTION
**Pull request by clrobinson-traction**
### Description

Adds `AuditLog` support to `openapi.yml`.

### Notes

When copy-pasting into ApiCurio, pay special attention to the examples of AuditLogs. The line `field_value_before: null` seems to be removed. The line can be found by searching for the "flag_enabled" example, in `audited_changes` of an audit log.

This may be a bug in ApiCurio.

### CODE REVIEW

**Comment by clrobinson-traction** [comment]
Thoughts: should we include (iso8601) `timestamp` as a format for an `AuditLogChange`?

File: openapi.yml [hunk 1]
```diff
@@ -2575,6 +2577,219 @@ paths:
...
+        summary: Path used to manage the list of AuditLogs.
+        description: >-
+            The REST endpoint/path used to list zero or more `AuditLog` entities.  This path contains a `GET`
+            operation to perform the list task.
+        get:
+            tags:
+                - AuditLogs
+            parameters:
+                -
+                    name: include
```
Comment (brandonjmckay) [inline openapi.yml]: Do AuditLogs have any relationships that we're going to show in the serializer?

Comment (clrobinson-traction) [reply openapi.yml]: I don't think so. They're pretty standalone.

Comment (brandonjmckay) [reply openapi.yml]: Should remove the `include` param then

**Comment by clrobinson-traction** [comment]
Closing this PR because the API documentation is likely to change as the Audit Log project develops.

**Pull request closed without merging**